tests/components/downloader/test_init.py

```diff
@@ -1,29 +1,15 @@
 """Tests for the downloader component init."""
 
-from unittest.mock import patch
+import pytest
 
-from homeassistant.components.downloader.const import (
-    CONF_DOWNLOAD_DIR,
-    DOMAIN,
-    SERVICE_DOWNLOAD_FILE,
-)
+from homeassistant.components.downloader.const import DOMAIN, SERVICE_DOWNLOAD_FILE
 from homeassistant.config_entries import ConfigEntryState
 from homeassistant.core import HomeAssistant
 
-from tests.common import MockConfigEntry
 
-
-async def test_initialization(hass: HomeAssistant) -> None:
+@pytest.mark.usefixtures("loaded_config_entry")
+async def test_initialization(hass: HomeAssistant, loaded_config_entry) -> None:
```

Comment:
You only need to use `pytest.mark.usefixtures` if it's not in the arguments.
```suggestion
from tests.common import MockConfigEntry


async def test_initialization(hass: HomeAssistant, loaded_config_entry: MockConfigEntry) -> None:
```